### DISCUSSION TITLE
How to make tailwind intellisense work inside variants object.

### DISCUSSION THREAD

**Discussion by lorcan-codes** [joe-bell/cva#269] [Q&A] [answered]
Hi, thank you for creating this library.

I followed [this instruction](https://cva.style/docs/getting-started/installation) to make tailwind intelissense work with cva.

While it is working for cva simple inputs such as cva("tailwind classes here") and cva(["tailwind",  "classes,  "here"]). The intellisense is not working inside variants. For example I don't get any intellisense inside `variant.h1` below.

```js
cva("text-foreground", {
    variants: {
        variant: {
            h1: "font-inter font-extrabold leading-[var(--line-height-h1)] tracking-[length:var(--letter-spacing-h1)]",
        }
    },
})
```  

Is there any way to make intellisense work inside variants configuration?

**Comment by Arcrueid** [comment]
Maybe it's not the best solution, but so far it works for me.

```json
"tailwindCSS.experimental.classRegex": [
    ["cva\\(([\\s\\S]*?)\\)(;|\\n)", "[\"'`]([^\"'`]*).*?[\"'`]"],
]
```

**Comment by henrikzabel** [comment]
This is not working for me. Is there any other way?

**Comment by joe-bell** [comment, marked as answer]
Should be fixed with https://github.com/joe-bell/cva/pull/332
